etc/hub.api.md

```diff
@@ -69,10 +69,30 @@ export type APISig = {
     msg: string;
 };
 
+// @public (undocumented)
+export type ArchiveDealInfo = {
+    proposalCid: string;
+    miner: string;
+};
+
+// @public (undocumented)
+export type ArchiveInfo = {
+    key: string;
+    cid?: string;
+    deals: Array<ArchiveDealInfo>;
+};
+
 export { ArchiveInfoResponse }
 
 export { ArchiveResponse }
 
+// @public (undocumented)
+export type ArchiveStatus = {
+    key: string;
+    status: number;

```

Comment:
where can i get the status number mapping @asutula ?

Reply:
Its in the generated proto code I guess you're generating these `ArchiveStatus` objects from. Looks like:
```
export namespace ArchiveStatusResponse {
  export type AsObject = {
    key: string,
    status: ArchiveStatusResponse.StatusMap[keyof ArchiveStatusResponse.StatusMap],
    failedMsg: string,
  }

  export interface StatusMap {
    STATUS_UNSPECIFIED: 0;
    STATUS_EXECUTING: 1;
    STATUS_FAILED: 2;
    STATUS_DONE: 3;
    STATUS_CANCELED: 4;
  }

  export const Status: StatusMap;
}```

Reply:
oop, missed that.